### PR TITLE
Add support for Travis CI

### PR DESCRIPTION
**Pull request by PamelaM**
This PR adds support for Travis CI.  It's currently running in the Addgene account.

See my PR comments for details

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ django_datatable_view.egg-info
 .tm_properties
 db.sqlite3
 appengine
+.idea
```
Comment (PamelaM) [inline .gitignore]: I'm using PyCharm and need to prevent polluting this repo

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
+env:
+  - DJANGO=1.4.20
```
Comment (PamelaM) [inline .travis.yml]: I originally included 1.2 and 1.3, but they failed pretty spectacularly.
1.7 and 1.8 are also failing but fixing 1.7 compatibility is outside of the scope of *this* PR ('though it *does* let us see that 1.7 and 1.8 have 'issues')

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - DJANGO=1.6.11
+  - DJANGO=1.7.8
+  - DJANGO=1.8.2
+install:
+  - pip install -q Django==$DJANGO --use-mirrors
+  - pip install pep8 --use-mirrors
+  - pip install pyflakes
+  - pip install -q -e . --use-mirrors
+#before_script:
+  #- "pep8 --exclude=migrations --ignore=E501,E225 datatableview"
```
Comment (PamelaM) [inline .travis.yml]: Not willing to pep8-ify or pyflakes the source tree, but this is here for possible reference.

**Comment by tiliv** [comment]
Thanks for this!  I'm going to take a day in the middle of this week and really get these pull requests finalized.  Merges are coming!

**Pull request merged**